### DISCUSSION TITLE
[Web] Typing has huge delay in a document with many lines, especially on firefox

### DISCUSSION THREAD

**Discussion by Wizzel1** [singerdmx/flutter-quill#509] [Web]
To reproduce:

1. open the quill editor in Firefox.
2. Go to https://loremipsum.de/index_en.html and copy 200 paragraphs.
3. Paste them in the editor. 
4. Try to type text in the Editor.

Is there anything I can do to remove or at least reduce the lag when typing?

**Comment by edityomurti** [comment]
Any ideas on how to solve this problem?
